README.MD

```diff
@@ -9,286 +9,91 @@
 [![MIT license](https://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
 ![Release](https://img.shields.io/github/release/Microsoft/presidio.svg)
 
-## Description
+## What is Presidio
 
-Presidio *(Origin from Latin praesidium ‘protection, garrison’)* helps to ensure sensitive text is properly managed and governed. It provides fast ***analytics*** and ***anonymization*** for sensitive text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers and financial data.
+Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ helps to ensure sensitive text is properly managed and governed. It provides fast **_analytics_** and **_anonymization_** for sensitive text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers and financial data.
 Presidio analyzes the text using predefined or custom recognizers to identify entities, patterns, formats, and checksums with relevant context. Presidio leverages docker and kubernetes for workloads at scale.
 
-### Why use presidio?
-
 Presidio can be integrated into any data pipeline for intelligent PII scrubbing. It is open-source, transparent and scalable. Additionally, PII anonymization use-cases often require a different set of PII entities to be detected, some of which are domain or business specific. Presidio allows you to **customize or add new PII recognizers** via API or code to best fit your anonymization needs.
 
 :warning: Presidio can help identify sensitive/PII data in un/structured text. However, because Presidio is using trained ML models, there is no guarantee that Presidio will find all sensitive information. Consequently, additional systems and protections should be employed.
 
-## Demo
-
-[Try Presidio with your own data](https://aka.ms/presidio-demo)
-
-## Features
-
-***Unstructured text anonymization***
+[Presidio Features](docs/overview.md#features)
```

Comment:
The overview file is quite old, can we use the main README instead? I think they have a lot of overlapping content

Reply:
IMHO the high-level features should be on the main Readme

Reply:
is the overview file needed? I didn't see it's reference anywhere. Whated to reduce the amount of information in the main readme as it is hard to find information that way. I suggest updating the overview doc with most recent info and still referencing to it. WDYT?

Reply:
I think it can be deleted if it's not referenced anywhere

Reply:
still debating this :) will update

Reply:
@omri374 did some more refactoring and kept the overview.md, it is referenced in the index and well as the main readme